lib/Sapi.php

```diff
@@ -71,6 +71,11 @@ static function sendResponse(ResponseInterface $response) {
         $body = $response->getBody();
         if (is_null($body)) return;
 
+        if (is_callable($body)) {
+            $body();
```

Comment:
i'm not sure if it's necessary - i think that the same layer which is setting the headers (e.g. content length) is also setting the body (callback), so it can also pass the content length to the callback (e.g. `function() use ($contentLength) {...}`